_pytest/python.py

```diff
@@ -928,7 +928,12 @@ def _find_parametrized_scope(argnames, arg2fixturedefs, indirect):
 
 def _idval(val, argname, idx, idfn, config=None):
     if idfn:
-        s = idfn(val)
+        s = None
+        try:
+            s = idfn(val)
+        except Exception:
+            import warnings
+            warnings.warn("Raised while trying to determine id of parameter %s at position %d" % (argname, idx))
```

Comment:
Don't you guys think it'd be better to create an issue for this? Otherwise it may be forgotten when version 4.0 is finally developed.